docs/usage.md

```diff
@@ -118,11 +118,121 @@ directory.
 
 This means that analysis of these files failed and there is no Clang Static Analyzer output for these compilation commands.
 
-### Showing new bugs in your code after local edit
+
+## Step5: Integrate CodeChecker into your CI loop
+
+This section describes a recommended "way of working" about how CodeChecker is designed to be used
+in a CI environment to
+* generate daily report summaries
+* implement CI guard to prevent the introduction of new bugs into the code-base.
+
+In CodeChecker each bug has a unique hash identifier that is independent of the exact line of code 
+therefore resistant to shifts in the source code.
+With this feature CodeChecker can recognize the same and new bugs in two different version of the same source file.
+
+**In summary:**
+* Store daily runs of a module every day in a new run post-fixed with date. You can query  *new* and *resolved* bugs
+ using the [``cmd compare``](docs/user_guide.md#show-differences-between-two-runs-diff) or the WEB GUI.
+* Create a single run for each module in each branch and keep it up to date with code changes (commits). The CI
+ loop then can compare pull requests (commit attempts) against this run and list *new* bugs in the changed code. 
+ Programmers can also compare their local edits to this run to see if they would introduce any new issues.
```

Comment:
I think we should highlight here more that they should not compare to the daily runs.